openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/Permission.spec.ts

```diff
@@ -251,7 +251,9 @@ test('Permissions', async ({ userPage, adminPage }) => {
 
     await userPage.getByTestId('profiler').click();
     const testCaseResponse = userPage.waitForResponse(
-      '/api/v1/dataQuality/testCases/search/list?fields=*'
+      (response) =>
+        response.url().includes('/api/v1/dataQuality/testCases/') &&
```

Comment:
@ShaileshParmar11, I believe we should include `/search` as well in the url so that it specifically checks for that.

Reply:
The main issue is the difference between the main version and 1.5.5. In the main version, we have added search functionality which uses the `/dataQuality/testCases/search/list` API. However, in version 1.5.5, we don't have that functionality yet. That's why we need to take this approach.